test/ccc/core/base/panel/legend/legend-panel-spec.js

```diff
@@ -0,0 +1,111 @@
+define([
+  'ccc/pvc',
+  'ccc/def',
+  'test/utils',
+  'test/data-1'
+], function(pvc, def, utils, datas) {
+
+  describe("legend-panel -", function() {
+
+    describe("legendOverflow -", function() {
+      var size = {width: 50, height: 50};
+
+      describe("- Clip Mode", function() {
+
+        it("'legendItemCountMax' isn't taken into account in 'Clip Mode'", function() {
+          var chartOptions = createOptions({legendOverflow: 'clip', legendItemCountMax: 2});
+          var actual = createChart(chartOptions);
+
+          expect(chartOptions.legendItemCountMax < actual.legendItems).toBe(true);
```

Comment:
Please invert the order legendItems > legendItemCountMax and use the karma toBeGreaterThan operator.